projects/plugins/jetpack/extensions/blocks/subscriptions/index.js

```diff
@@ -17,48 +18,59 @@ import { getIconColor } from '../../shared/block-icons';
 export const name = 'subscriptions';
 export const icon = (
 	<SVG width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
-		<Rect
-			x="4.75"
-			y="6.75"
-			width="14.5"
-			height="10.5"
-			rx="1.25"
-			stroke={ getIconColor() }
-			strokeWidth="1.5"
-			fill="none"
-		/>
 		<Path
-			d="M19 7L13.3609 12.2363C12.5935 12.9489 11.4065 12.9489 10.6391 12.2363L5 7"
-			stroke={ getIconColor() }
-			strokeWidth="1.5"
-			strokeLinejoin="bevel"
-			fill="none"
+			d="M20.9997 6.96749L21 7V17C21 18.1046 20.1046 19 19 19H5C3.89543 19 3 18.1046 3 17V7L3.00026 6.96747L3 6.96092C3 5.87793 3.87793 5 4.96092 5H19.0391C20.1221 5 21 5.87793 21 6.96092L20.9997 6.96749ZM19.3044 17.3967L13.9142 12.9048L13.2804 13.433C12.5387 14.0511 11.4613 14.0511 10.7196 13.433L10.0858 12.9048L4.69555 17.3967C4.77987 17.4615 4.88543 17.5 5 17.5H19C19.1146 17.5 19.2201 17.4615 19.3044 17.3967ZM19.5 15.6071V8.25L15.0858 11.9285L19.5 15.6071ZM4.5 8.25V15.6071L8.91424 11.9285L4.5 8.25ZM4.79543 6.54363C4.85788 6.51559 4.92712 6.5 5 6.5H19C19.0729 6.5 19.1421 6.51559 19.2046 6.54363L12.3201 12.2807C12.1347 12.4352 11.8653 12.4352 11.6799 12.2807L4.79543 6.54363Z"
+			fill={ getIconColor() }
+			fillRule="evenodd"
+			clipRule="evenodd"
 		/>
 	</SVG>
 );
 export const settings = {
-	title: __( 'Subscription Form', 'jetpack' ),
+	title: __( 'Subscribe', 'jetpack' ),
 	description: (
-		<p>
-			{ __(
-				'A form enabling readers to get notifications when new posts are published from this site.',
-				'jetpack'
-			) }
-		</p>
+		<>
+			<p>
+				{ __(
+					'Allow readers to receive a newsletter with future posts in their inbox.',
+					'jetpack'
+				) }
+			</p>
+			<p>
+				{ createInterpolateElement(
+					__(
+						'Subscribers can get notifications through email or <ExternalLink>the Reader app</ExternalLink>.',
+						'jetpack'
+					),
+					{ ExternalLink: <ExternalLink href={ 'https://wordpress.com/read' } /> }
+				) }
+			</p>
+		</>
 	),
 	icon: {
 		src: icon,
 		foreground: getIconColor(),
 	},
 	category: 'grow',
 	keywords: [
-		_x( 'subscribe', 'block search term', 'jetpack' ),
-		_x( 'join', 'block search term', 'jetpack' ),
+		_x( 'newsletter', 'block search term', 'jetpack' ),
+		_x( 'email', 'block search term', 'jetpack' ),
 		_x( 'follow', 'block search term', 'jetpack' ),
 	],
 	attributes,
 	edit,
 	save,
+	styles: [
+		{
+			name: 'compact',
+			label: __( 'Compact', 'jetpack' ),
+			isDefault: true,
+		},
+		{
+			name: 'split',
+			label: __( 'Split', 'jetpack' ),
+		},
+	],
```

Comment:
These settings do not seem to do anything in my tests:

https://user-images.githubusercontent.com/426388/154103676-bd932fbf-1a36-4a38-b216-5e95db7296c8.mov

Reply:
Which theme?

Reply:
This was happening with Twenty Twenty as well as Twenty Twenty Two iirc.

Reply:
I see that your Styles panel has a preview on it. Is this WPCOM specific? Going to test this on WPCOM next. In the meantime, Jetpack via Jurassic is fine for 2020, 2021 & 2022.

**Twenty-Twenty Two**
https://user-images.githubusercontent.com/1287077/154233154-9279fc59-21b7-4efd-a256-440077b47636.mp4

**Twenty-Twenty One**
https://user-images.githubusercontent.com/1287077/154233216-036ddc0e-921d-4b54-9928-842a668c9262.mp4

**Twenty-Twenty**
https://user-images.githubusercontent.com/1287077/154233380-e5148559-3c3e-4917-b348-de2c41e9ede2.mp4

Reply:
The video has a relatively old UI.

Reply:
I'll try again tomorrow. 👍

Reply:
I still run into this, on a site with the Twenty Twenty theme and running WP 5.8.3 (Jetpack supports current WP -1)

Reply:
Just in case, Compact and Split should have no visible effect if there is also a "new line" forced through the setting.